### PR TITLE
DBus + newer Glib + autoconf-archive + pango

### PR DESCRIPTION
**Pull request by flyingP0tat0**
I added DBus and autoconf-archive packages and modified Glib to use a newer version.

### CODE REVIEW

File: packages/autoconf_archive.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Autoconf_archive < Package
+  version '2017-03-21'
+  source_url 'https://ftp.wrz.de/pub/gnu/autoconf-archive/autoconf-archive-2017.03.21.tar.xz'
```
Comment (jam7) [inline packages/autoconf_archive.rb]: I think it's better to use public url `http://ftpmirror.gnu.org/autoconf-archive/autoconf-archive-2017.03.21.tar.xz` as described in https://www.gnu.org/software/autoconf-archive/Downloads.html.  This ftpmirror.gnu.org redirects us to the nearest mirror ftp site.

Comment (flyingP0tat0) [inline packages/autoconf_archive.rb]: Yes, my fault. Changed it.

**Comment by flyingP0tat0** [comment]
Added some more packages, pango and its dependencies.

**Comment by flyingP0tat0** [comment]
Alright, I feel like this pull request is somewhat unorganized. I will close this one and reopen multiple specific to the packages added.

**Pull request closed without merging**